src/test/java/steps/AuthSteps.java

```diff
@@ -22,16 +22,15 @@
 
 public interface AuthSteps {
   static NavigationPage login(String login, String password) {
-    LoginPage lp = BasePage.getPage(NavigationPage.class)
+    BasePage.getPage(NavigationPage.class)
         .navigateLogin()
         .openLoginPage();
-    lp.setUserName(login);
-    lp.setUserPassword(password);
-    return lp.submit();
+    BasePage.getPage(LoginPage.class).setUserName(login);
+    BasePage.getPage(LoginPage.class).setUserPassword(password);
+    return BasePage.getPage(LoginPage.class).submit();
   }
```

Comment:
Please change the return types of the methods from void to pages so that you could call them in chain